apps/backend/src/main/resources/application.yaml

```diff
@@ -1,6 +1,15 @@
 ktor:
-    application:
-        modules:
-            - org.eu.nl.ApplicationKt.module
-    deployment:
-        port: 8080
+  application:
+    modules:
+      - org.eu.nl.ApplicationKt.module
+#  development: true
+  deployment:
+#    host: localhost.api.craftsmans-ledger.net
+    port: 7200
+#    sslPort: 7200
+#  security:
+#    ssl:
+#      keyAlias: craftsmans-ledger-key
+#      keyStore: keystore.p12
+#      keyStorePassword: ${KEYSTORE_PASSWORD}
+#      privateKeyPassword: ${KEYSTORE_PASSWORD}
```

Comment:
These options that are commented out are what should be set when the `MODE` or `KTOR_ENV` is set to `development`